File: iniciante/1048/1048.py
# 1048 - Aumento de Salario - beecrowd
# Iniciante - Nivel 2

def calculoReajusteSalario(salario : float):
    tamanho = 4

    SALARIO = [400, 800, 1200, 2000]
    PERCENTUAL = [15, 12, 10, 7]

    resultadoReajuste = 4

    for i in range(tamanho):
        if salario <= SALARIO[i]:
            resultadoReajuste = PERCENTUAL[i]
            break

    novoSalario = salario * (1 + (resultadoReajuste / 100))

    print(f"Novo salario: {novoSalario:.2f}")
    print(f"Reajuste ganho: {novoSalario - salario:.2f}")
    print(f"Em percentual: {resultadoReajuste} %")

def main():
    salario = float(input())
    calculoReajusteSalario(salario)

if __name__ == "__main__":
    main()

File: iniciante/1048/1048.c
// 1048 - Aumento de Salario - beecrowd
// Iniciante - Nivel 2

#include <stdio.h>

void calculoReajusteSalario(double salario)
{
    const int tamanho = 4;

    const double SALARIO[] = {400, 800, 1200, 2000};
    const int PERCENTUAL[] = {15, 12, 10, 7};

    int resultadoPercentual = 4;

    for (int i = 0; i < tamanho; i++)
    {
        if (salario <= SALARIO[i])
        {
            resultadoPercentual = PERCENTUAL[i];
            break;
        }
    }

    double novoSalario = salario * (1 + (resultadoPercentual / 100.0));

    printf("Novo salario: %.2lf\n", novoSalario);
    printf("Reajuste ganho: %.2lf\n", novoSalario - salario);
    printf("Em percentual: %d %%\n", resultadoPercentual);
}

int main()
{
    double salario;
    scanf("%lf", &salario);

    calculoReajusteSalario(salario);

    return 0;
}
